.gitignore

```diff
@@ -2,6 +2,9 @@
 .DS_Store
 Thumbs.db
 
+# Generated files from GraphViz
+prereq_flowchart/TEMP_graphviz/*
```

Comment:
why does it generate files inside our python package?

Reply:
i think it would be better to generate files in the repo root, perhaps in like `out/diagrams` or something (where `out/` is gitignored)

Reply:
Sounds good, I'll base the file location relative to the file itself.